src/integrationTest/resources/log4j2-test.properties

```diff
@@ -19,3 +19,5 @@ logger.testsecconfig.name=org.opensearch.test.framework.TestSecurityConfig
 logger.testsecconfig.level = info
 logger.localopensearchcluster.name=org.opensearch.test.framework.cluster.LocalOpenSearchCluster
 logger.localopensearchcluster.level = info
+logger.eveluator.name=org.opensearch.security.privileges.PrivilegesEvaluator
```

Comment:
Was this intentionally left enabled?  If so it would be useful to add a comment for how/why its enabled

Reply:
This is unnecessary reminder after some problem investigation, removed